src/LodeRunner.Data/CosmosDBRepository.cs

```diff
@@ -227,8 +227,14 @@ public async Task<bool> CreateClient()
 
                 return true;
             }
-            catch (Exception)
+            catch (CosmosException ex)
```

Comment:
Let's check for the specific code that indicates that it can't connect due to auth or network.  Looking at the [HttpStatusCodes](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=net-6.0) it is probably ones like:
- 502 - BadGateway
- 403 - Forbidden
- 504 - GatewayTimeout
- 401 - Unauthorized

Those above are probably good status codes to tell folks to check their connection, firewall, and privileges :)